frontend/packages/dev-console/src/components/add/AddCardSectionSkeleton.scss

```diff
@@ -0,0 +1,35 @@
+.odc-add-section-skeleton {
+  position: relative;
+  width: 100%;
+  &__background {
+    z-index: 1;
```

Comment:
`z-index` is not required if you remove both `z-index`, or?

Reply:
z-index is not required. I added z-index so as to not rely on default.
but i think we can remove it.